agent/command/papertrail_trace_test.go

```diff
@@ -213,6 +213,184 @@ func TestPapertrailTrace(t *testing.T) {
 	}
 }
 
+func TestPapertrailGetFiles(t *testing.T) {
+	type testFile struct {
+		path    string
+		content string
+		want    papertrailTraceFile
+	}
+
+	cases := map[string]struct {
+		files    []testFile
+		patterns []string
+	}{
+		"simple": {
```

Comment:
nit: The Evergreen style for tests is to use `PascalCase` not dashes + all lowercase.

Reply:
I changed all test case named to be PascalCase